src/modules/ROOT/pages/changelog/202303/02.json

```diff
@@ -0,0 +1,233 @@
+{
+  "meta": {
+    "releasedOn": "2023.06.08"
+  },
+  "added": [
+    {
+      "what": [
+        "option for transparent color in the color picker."
+      ],
+      "why": [
+        "to give users more configuration options."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1211"
+      ]
+    },
+    {
+      "what": [
+        "the option to set a record list filter as a preset filter for a user."
+      ],
+      "why": [
+        "to give the user the ability to filter items in a record list before the list is loaded."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1227/commits/aa28be29cba69efce55028fa08c3a6c9a45ccc3d"
+      ]
+    },
+    {
+      "what": [
+        "menu with 'more' options (delete, un-delete and permissions) to resources in {APP_NAME_COMPOSE}, {APP_NAME_ADMIN}, {APP_NAME_REPORTER}, {APP_NAME_WORKFLOW}."
+      ],
+      "why": [
+        "so the user doesn't have to scroll horizontally to access all resource button/options."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/46f5455bd0ae67b9de2159f9d04b054a711832e1"
+      ]
+    },
+    {
+      "what": [
+        "page layouts to namespace export resources."
```

Comment:
I'd say this was more of a fix rather than an addition; up to you where you want to put it.
In any case, do explain the point of this addition/fix; what problem was the user facing -- when exporting namespaces, page layouts weren't included because ...